### PR TITLE
New lambdas for SAR Enhancement Mar-2019

### PR DESCRIPTION
**Pull request by kishor-bhatt**
This request is for enhancements of updating AWS SAAS URL, readme files and other content changes. Changes Confluence Page - https://confluence.in.here.com/display/DeveloperRelations/Enhancement+Product+Requirements

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,4 +60,8 @@ typings/
 *-packaged.yml
 .vs
 bin
-obj
\ No newline at end of file
+obj
+
+# User-specific stuff:
+*.idea/*
+*.DS_Store
```
Comment (sschuberth) [inline .gitignore]: Nit: Please add a final new line.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # AWS SAM for HERE Maps APIs
 ## Introduction
-This project provides [AWS Lambdas](https://aws.amazon.com/lambda/) as __proxies__ for several of the [HERE Maps APIs](https://developer.here.com/). These AWS Lambdas are packaged as per the [AWS Serverless Application Model](https://aws.amazon.com/about-aws/whats-new/2016/11/introducing-the-aws-serverless-application-model/).
+This project provides [AWS Lambdas](https://aws.amazon.com/lambda/) as __proxies__ for several of the [HERE Location Services APIs](http://saas.awsmarketplace.here.com/rest-api). These AWS Lambdas are packaged as per the [AWS Serverless Application Model](https://aws.amazon.com/about-aws/whats-new/2016/11/introducing-the-aws-serverless-application-model/).
 
-�AWS SAM is natively supported by AWS CloudFormation and defines simplified syntax for expressing serverless resources. The specification currently covers APIs, Lambda functions and Amazon DynamoDB tables.�
+�AWS SAM is natively supported by AWS CloudFormation and defines simplified syntax for expressing serverless resources. The specification currently covers APIs, Lambda functions and Amazon DynamoDB tables.�
```
Comment (sschuberth) [inline README.md]: Looks like there's some encoding issue being introduced. Please ensure to use UTF-8 encoding.

File: serverlessFunctions/geocode/geocode.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/geocode/geocode.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/geocode/geocodesuggest.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/geocode/geocodesuggest.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/mapimage/mapimage.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/mapimage/mapimage.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/places/places.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/places/places.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/routing/routing.js [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/routing/routing.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/routing/waypointseq.js [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/routing/waypointseq.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/tollcost/tollcost.js [hunk 9]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/tollcost/tollcost.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/traffic/traffic.js [hunk 10]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/traffic/traffic.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/transit/transit.js [hunk 11]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/transit/transit.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

File: serverlessFunctions/weather/weather.js [hunk 12]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```
Comment (sschuberth) [inline serverlessFunctions/weather/weather.js]: Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: serverlessFunctions/geocode/geocode.js (hunk 3 above)
Comment (kishor-bhatt) [reply serverlessFunctions/geocode/geocode.js]: Ok, copyright header should be left to 2017 or modified to 2017-2019 for all files ?

**Review by palermo4 — APPROVED** [review]
Reviewed and approved

**Comment by palermo4** [comment]
New lambdas and fixes

**Pull request merged**